cluster-up/cluster/kind-1.19-vgpu/README.md

```diff
@@ -0,0 +1,43 @@
+# K8S 1.19.0 with mdev support in a Kind cluster
+
+Provides a pre-deployed k8s cluster with version 1.19.0 that runs using [kind](https://github.com/kubernetes-sigs/kind) The cluster is completely ephemeral and is recreated on every cluster restart. 
+The KubeVirt containers are built on the local machine and are then pushed to a registry which is exposed at
+`localhost:5000`.
+
+## Bringing the cluster up
+
+```bash
+export KUBEVIRT_PROVIDER=kind-1.19.0-vgpu
+make cluster-up
```

Comment:
It might be good to comment here that you need to execute as root.

Reply:
Done